public/js/main.js

```diff
@@ -645,6 +645,33 @@ function alignRightHeaders() {
     });
 }
 
+function addSingleQuoteToCsvIfNeeded(str) {
+  const specialChars = ['=', '+', '-', '@', '\t', '\r'];
+  if (specialChars.some(char => str.startsWith(char))) {
+    return "'" + str;
+  } else {
+    return str;
+  }
+}
+
+/**
+ * Patch the Tabulator CSV downloader logic to add a single quote to values starting with special characters
+ * This fixes the issue where Excel would interpret these values as formulas, creating a security risk
+ */
+function patchTabulatorCsvDownloader() {
+  const getFieldValueCopy = Tabulator.prototype.moduleBindings?.download?.prototype?.getFieldValue;
+  if (getFieldValueCopy === undefined) {
+    console.error('Ooops, something is wrong. CSV download might not work correctly. LKE-4315');
```

Comment:
I think we need to remove the ticket number from the error msg as the plugin is used by client

Reply:
Yes, I know but I did it on purpose. It should be spotted during development. Worst case scenario, it's a JIra ticket number, no security risk and the customer would probably see this as an error code. He would send it to use which would help the investigation (this is the intent).
Most of the time, the dev working on the new code would see it and fix it before prod.
wdyt? I'm ok to remove if you still prefer